src/components/Body/MovieCrads/MovieCard/MovieCard.tsx

```diff
@@ -10,22 +12,30 @@ type MoviePropsType = {
 };
 
 function MovieCard({ movie }: MoviePropsType) {
-  const { selectMovie } = useMoviesContext();
+  const dispatch = useAppDispatch();
+
+  const handleMovieClick = () => {
+    dispatch(openMovieDetails());
+    dispatch(selectMovie(movie.id));
+  };
 
   return (
     <div className={styles.movieCard}>
       <div className={styles.movieCard_settings}>
         <MovieSettings />
       </div>
-      <div
-        className={styles.movieCard_imageWrapper}
-        onClick={() => selectMovie(movie)}
-      >
-        <img className={styles.movieCard_image} src={movie.url} alt="Url" />
+      <div className={styles.movieCard_imageWrapper} onClick={handleMovieClick}>
+        <img
+          className={styles.movieCard_image}
+          src={movie.poster_path}
+          alt="Url"
+        />
```

Comment:
You should use onError attribute if there is no image at this url

The user must not see this:
![image](https://user-images.githubusercontent.com/106384184/209788395-7c63793b-088a-4b61-8b6e-d8392df28582.png)

alt attribute should contain meaningful description, please, use movie's name

It would be nice to have separated component for image `Poster` and use this component on movies details page and for Card component